Documentation/Tutorials/Avrorouter-Tutorial.md

```diff
@@ -125,7 +125,7 @@ the following format:
 ```
 
 The avrorouter uses the schema file to identify the columns, their names and
-what type they are. The Name fiels contains the name of the column and the Type
+what type they are. The Name fields contains the name of the column and the Type
```

Comment:
I think this was intended to be `field` since the Avro specification has a `field` parameter in the schema definition of a record.